engine/src/test/java/io/camunda/zeebe/engine/util/RecordingJobStreamer.java

```diff
@@ -55,6 +56,14 @@ public record TestActivationProperties(
 
     @Override
     public void wrap(final DirectBuffer buffer, final int offset, final int length) {}
+
```

Comment:
:wrench: We can consider removing the `TestActivationProperties` class and use the proper `JobActivationPropertiesImpl` in `ActivatableJobsPushTest#setUp()` (it's only used there, so it's not a large effort.

This way, we ensure that the actual `JobActivationProperties` implementation is covered by our unit tests.